src/main/groovy/org/pih/warehouse/inventory/DiscrepancyCause.groovy

```diff
@@ -0,0 +1,9 @@
+package org.pih.warehouse.inventory
+
+enum DiscrepancyCause {
```

Comment:
I'm realizing that we never had a group discussion on what values should exist here (I wrote these ones myself). Lets discuss it with Manon.

We'll also need to localize each of these.